jre/guide/java/java_jre_deployment_config_configured/java_jre_deployment_config_exists/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-title: 'The Java Configuration File Exists'
+title: 'Oracle JRE 8 must have a deployment.config file present'
```

Comment:
This change doesn't need to take place.

Reply:
These titles were changes to align with the title in the STIG.

Reply:
I know you want to make a literal 1:1 to the STIG, but these type of changes are regressions as there are different types of java that these rules map to. In other words, this rule is mapped to IBM, Oracle, and OpenJDK making this change a bug.

Reply:
This makes perfect sense. Thank you!